server/service/src/sync/translations/invoice.rs

```diff
@@ -733,6 +725,10 @@ fn map_legacy(
         .confirm_date
         .map(|confirm_date| NaiveDateTime::new(confirm_date, data.confirm_time));
 
+    let finalised_datetime = data
+        .finalised_date
+        .map(|finalised_date| NaiveDateTime::new(finalised_date, data.finalised_time));
```

Comment:
I'm wondering if we should fallback to confirmed time if the invoice is finalised? I think we can't guarantee that the date is populated :S.

Aside, the ledger fixes essentially use `confirmed_date` anyway, so I'm not sure anything really breaks if finalised date is wrong or None on a finalised invoice.